### DISCUSSION TITLE
CrmSvcUtil.exe as dotnet tool

### DISCUSSION THREAD

**Discussion by hivanov** [microsoft/PowerPlatform-DataverseServiceClient#194] [Ideas]
Would it be possible to port CrmSvcUtil.exe as a dotnet command-line tool? I'm especially interested in running the tool under Linux, since currently my build requires a Windows installation, .net 5 SDK, and the CrmSvcUtil downloaded as powershell script, before I'm able to run it.

This would be useful even if you are not able to port the entirety of the CrmSvcUtil (e.g., the interactive login part).

**Comment by MattB-msft** [comment]
This is being explored actively as we look to bring the tool into the PowerPlatform CLI.
